server/liveblog/blogs/blogs.py

```diff
@@ -34,13 +53,31 @@ class BlogsResource(Resource):
             'type': 'string',
             'allowed': ['blog'],
             'default': 'blog'
+        },
+        'cid': {
+            'type': 'integer'
         }
     }
 
+    item_methods = ['GET', 'PATCH', 'PUT', 'DELETE']
     privileges = {'GET': 'blogs', 'POST': 'blogs', 'PATCH': 'blogs', 'DELETE': 'blogs'}
 
 
-class BlogService(BaseService):
+def set_cid_on_blogs(item):
+        cid = update_key('blog_cid', flag=True)
+        if cid:
+            item['cid'] = cid
+        return cid
+
+
+def get_last_cid(item):
+    cid = update_key('blog_cid', flag=False)
+    if cid:
+        item['cid'] = cid
```

Comment:
The item can't be none and it is strange since for me none of the tests are failing